03/app.js

```diff
@@ -5,14 +5,53 @@ const a = randomNumber(min, max);
 const b = randomNumber(min, max);
 const c = randomNumber(min, max);
 
-console.log(a, b, c);
-
-
-
-
-
-
+// console.log(a, b, c);
 
 function randomNumber(min, max) {
     return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+}
+
+function getSum(a, b, c) {
+    a = parseInt(a);
+    b = parseInt(b);
+    c = parseInt(c);
+    const arr = [a, b, c]
```

Comment:
Nie zmieniamy wartości dla parametrów (nawet jeśli to tylko konwersja na liczbę) dlatego lepiej będzie tak: [parseInt(a),  parseInt(b),  parseInt(c)]